linkml_model/model/schema/aliases.yaml

```diff
@@ -0,0 +1,84 @@
+id: https://w3id.org/linkml/mappings
+title: LinkML Schema Mappings
+name: mappings
+description: LinkML model for mappings
+license: https://creativecommons.org/publicdomain/zero/1.0/
+version: 2.0.0
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  skos: http://www.w3.org/2004/02/skos/core#
+  OIO: http://www.geneontology.org/formats/oboInOwl#
+  IAO: http://purl.obolibrary.org/obo/IAO_
+
+default_curi_maps:
+    - semweb_context
+
+default_prefix: linkml
+default_range: string
+
+emit_prefixes:
+  - linkml
+  - rdf
+  - rdfs
+  - xsd
+  - skos
+  - OIO
+  - IAO
+
+imports:
+  - linkml:types
+  - linkml:meta
+
+slots:
+  alias_source:
+    range: string
+    required: true
+    description: >-
+      Can be a URIOrCurie that represents the source of the alias, or it can be a simple string that
+      describes the source of the alias.
+
+  alias_scope:
+    range: alias_scope_enum
+
+  alias:
+    domain: slot_definition
```

Comment:
not just for slots

Reply:
ah I see, it's already this way in the metamodel

alias is a very special purpose slot in the metamodel... unfortunately we have overloaded "alias" a lot...

Reply:
this is a slightly odd existing slot in the model already (I just moved it from meta.yaml to alias.yaml).  It's for setting a 'preferred label' of a slot.  alias_definition is what I named the 'alias class' in order to not break this model.